frontend/components/GitHubRepoLink.jsx

```diff
@@ -3,21 +3,36 @@ import PropTypes from 'prop-types';
 
 import GitHubMark from './GitHubMark';
 
-const GitHubRepoLink = ({ owner, repository }) => (
-  <a
-    className="repo-link"
-    href={`https://github.com/${owner}/${repository}`}
-    title="Visit repository"
-    target="_blank"
-    rel="noopener noreferrer"
-  >
-    <GitHubMark />
-  </a>
-);
+const GitHubRepoLink = ({ owner, repository, branch }) => {
+  let href = `https://github.com/${owner}/${repository}`;
+  let title = 'View repository';
+
+  if (branch) {
```

Comment:
modified this component to render a link to branch on github if the `branch` prop is supplied